src/package-manager.ts

```diff
@@ -105,13 +105,13 @@ export async function detectPackageManager(
                   (pm) => pm.name === name && pm.majorVersion === majorVersion,
                 ) || packageManagers.find((pm) => pm.name === name);
               return {
+                ...packageManager,
                 name,
                 command: name,
                 version,
                 majorVersion,
```

Comment:
If you check L97 we default version to `0.0.0`, the specific order was to avoid this (in case parser cannot find version) we should fix that